src/DetailsView/details-view-initializer.ts

```diff
@@ -749,18 +744,14 @@ if (tabId != null) {
             window.selfFastPass = selfFastPass;
         })
         .catch(() => {
-            const renderer = createNullifiedRenderer(
-                document,
-                ReactDOM.render,
-                createDefaultLogger(),
-            );
+            const renderer = createNullifiedRenderer(document, createRoot, createDefaultLogger());
             renderer.render();
         });
 }
 
 function createNullifiedRenderer(
     doc: Document,
-    render: typeof ReactDOM.render,
+    render: typeof createRoot,
```

Comment:
this change is part of React18, render is replaced with createRoot(container).render()